doc/CMakeLists.txt

```diff
@@ -3,8 +3,7 @@
 cmake_minimum_required(VERSION 3.20.0)
 project(Zephyr-Kernel-Doc LANGUAGES)
 
-set(NO_BOILERPLATE TRUE)
```

Comment:
If this is the last user of `NO_BOILERPLATE`, we can remove the check in `ZephyrConfig.cmake`?

Reply:
done